developers/integrations/compute-infrastructure/modal/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Modal
+sidebar_position: 1
+---
+
+[Modal](https://modal.com/) provides a serverless platform that enables developers to easily deploy and automatically scale applications with on-demand access to GPUs and a custom high-performance container runtime.
+
+## Modal and Weaviate
+Weaviate integrates with Modal by leveraging its serverless infrastructure for the fast generation of embeddings and generative model calls. The [Weaviate client can be hosted](https://modal.com/docs/examples/vector-analogies-wikipedia#deploy-a-serverless-read-only-weaviate-client-with-modal) on Modal to dynamically scale based on workload demands. 
+
+
+## Our Resources 
+**Hands on Learning**: Content framed to build your technical understanding with end-to-end tutorials.
+
+| Topic | Description | Resource | 
+| --- | --- | --- |
+| Embed and Search Text at Scale with Modal and Weaviate | Build a full application that discovers analogies between Wikipedia articles by combining serverless infrastructure from Modal with the search and storage capabilities of Weaviate. | [Blog post](/blog/modal-and-weaviate#modal-serverless-infrastructure-for-gpus-and-more), [Notebook](https://github.com/weaviate/recipes/tree/main/integrations/compute-infrastructure/modal), [Modal Guide](https://modal.com/docs/examples/vector-analogies-wikipedia#deploy-a-serverless-read-only-weaviate-client-with-modal)|
```

Comment:
Consider splitting into shorter sentences
Build a full application that discovers analogies between Wikipedia articles. Combine serverless infrastructure from Modal with the search and storage capabilities of Weaviate.